test/e2e-common.sh

```diff
@@ -210,6 +211,85 @@ function enable_monitoring(){
       --role roles/cloudtrace.agent
 }
 
+# test_authentication_check_for_brokercell tests the authentication check function for BrokerCell.
+# This test required the entire BrokerCell to be non-functional.
+# In order to avoid running in parallel with other tests, we put it in the shell script, rather than more common go tests.
+# TODO Once we support multiple BrokerCells with distinct authentication mechanisms, make this a normal test that runs in parallel with the others.
+function test_authentication_check_for_brokercell() {
+  echo "Starting authentication check test for brokercell."
+  local auth_mode=${1}
+  kubectl apply -f ${CONFIG_WARMUP_GCP_BROKER}
```

Comment:
Random nits: some shellcheck actually does `"${CONFIG_WARMUP_GCP_BROKER}"`.  It is not enforced at all, but I see you are using both with and without quotes in your code, so maybe it makes sense to always add that.

Reply:
The place where double quote was used is because it is caught by IDE automatic syntax checking for SC2086: Double quote to prevent globbing and word splitting. 
For this command, it seems it doesn't volatile that rule so it didn't show the warning to me...anyway, I will change it with double quote to make it consistent.